CMakeLists.txt

```diff
@@ -343,21 +344,31 @@ if(MZ_ZSTD)
         find_package(PkgConfig QUIET)
         if(PKGCONFIG_FOUND)
             pkg_check_modules(ZSTD libzstd)
+            if(ZSTD_FOUND)
+                list(APPEND MINIZIP_INC ${ZSTD_INCLUDE_DIRS})
+                list(APPEND MINIZIP_LIB ${ZSTD_LIBRARIES})
+                list(APPEND MINIZIP_LBD ${ZSTD_LIBRARY_DIRS})
+                set(PC_PRIVATE_LIBS "${PC_PRIVATE_LIBS} -lzstd")
+                list(APPEND MINIZIP_DEP_PKG ZSTD)
+            endif()
         endif()
         if(NOT ZSTD_FOUND)
             find_package(ZSTD QUIET)
-            set(ZSTD_VERSION ${ZSTD_VERSION_STRING})
+            if(ZSTD_FOUND)
+                set(ZSTD_VERSION ${ZSTD_VERSION_STRING})
+                if(TARGET libzstd_static)
```

Comment:
Why not do: 
```
set(ZSTD_LIBRARIES libzstd_static)
```
Then we can use:
```
list(APPEND MINIZIP_LIB ${ZSTD_LIBRARIES})
```
down below in `if(ZSTD_FOUND AND NOT MZ_FORCE_FETCH_LIBS)` like before?

Reply:
i can simplify this a bit, i just added this because I thought it would be pointless to add 
```cmake
                    set(PC_PRIVATE_LIBS "${PC_PRIVATE_LIBS} -lzstd")
                    list(APPEND MINIZIP_DEP_PKG ZSTD)
```
if we are linking statically, but after further thinking, it would fail if minizip is itself is static